bin/dfx-sns-demo

```diff
@@ -12,6 +12,7 @@ clap.define short=n long=network desc="The dfx network to use" variable=DFX_NETW
 clap.define short=c long=ic_commit desc="The IC commit to use" variable=DFX_IC_COMMIT default=""
 clap.define short=x long=ic_dir desc="Directory containing the ic source code" variable=IC_REPO_DIR default="$HOME/dfn/ic"
 clap.define short=y long=nd_dir desc="Directory containing the nns-dapp source code" variable=ND_REPO_DIR default="$HOME/dfn/nns-dapp"
+clap.define long=config_index desc="The initial value for the dfx-sns-config-random --config_index parameter" variable=CONFIG_INDEX default="0"
```

Comment:
What do you mean by "initial"? This is just *the* value, right?

Reply:
Updated.